### ISSUE TITLE
[Adaptive Cards - Product Video]: Screen Reader does not convey the descriptive information about the 'Learn More' button. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8799]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/explorer/TextBlock.html
OS Version: 23H2 (OS Build 26020.1000)
Browser Version: Version 122.0.2348.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open above URL with valid credential.
2. Navigate to 'Samples' link and press enter key to activate it.
3. Select the 'Product Video' option and press enter key to activate it.
4. Navigate to 'Learn More' button and observe the issue.

**Actual Result:**
While navigating to the product video screen, screen reader is not conveying the descriptive information for the 'Learn More' button. It is announcing as only **'Link Learn More'.**

**Expected Result:**
Screen Reader should convey the descriptive information for the 'Learn More' and it should announce as **'Learn More for Adaptive Card Product Video button'.**

**Note:**
Same issue is observed with NVDA.

**User Impact:**
It will impact the screen reader user as they will not able to know the correct information about the Learn more button also they will not able to know actual purpose of Learn more.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

“ Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section”

### Screenshots

![WCAG1 3 1- Learn more button should be more descriptive](https://github.com/microsoft/AdaptiveCards/assets/95913748/dc8242b9-516a-45a4-a2cb-c67e2e6698ce)

https://github.com/microsoft/AdaptiveCards/assets/95913748/61879ab2-e635-4785-93fe-6478a36f70ec

### Card JSON

```JSON
NA
```
